cmd/registry-server/main.go

```diff
@@ -93,30 +94,37 @@ func main() {
 	pflag.StringVarP(&configPath, "configuration", "c", "", "The server configuration file to load.")
 	pflag.Parse()
 
+	bootLogger := log.NewLogger()
```

Comment:
Guessing that "bootLogger" means that this is only used during startup of the server, and that this is a separate logger because it can't be configured before it is needed.